Adds Brig Physicians

nmajask opened the pull request:
# Github documenting your Pull Request

Adds a new job, a Brig Physician. Their job is to treat prisoners and officers, preform surgeries, recover dead officers, and generally help sec with medical issues. Basically, they are a mining medic but for sec. While they have access to the brig and have a flash they are **not a security officer** and should help officers, not do their job for them. They can not arrest people or hunt for antags unless the situation is dire (ie nukies are attacking). They should follow the orders of the HoS and warden when it comes to security matters, and the CMO when it comes to medical ones. This job is based on the one on [Bee Station](https://wiki.beestation13.com/view/Brig_Physician), so look to that for a more in depth description of the job.

## Why?

1. Many times I or other people have wanted to work the infirmary in the brig
2. There are many reasons sec would need medical staff in the brig, giving you plenty to do (implant removal, pacifying people, giving officers cybernetics, treating people, ect.)
3. Could serve as a nice way for people who normally play medical to try working with sec
4. Could help reduce the stress of sec, as there would be more people to help you
5. Brig Physicians have been jobs on other servers, like Beestation

## Job Details

Here is an overview of the attributes

- By default, only one slot is open round start 
- They have access to most of the brig _and perma_, the access that mining medics have for medical, and (on skeleton crew) _access to the sec office_ Removed as requested by adamsong
- It is part of sec on the occupation select menu
- Part of medsci in game and is payed from the medical budget
- The heads which the brig physician are under is the CMO _and HoS_ Removed as requested by adamsong
- They require a character to be at least 24 (like most med staff)

One issue with the Brig Physician is how does it work with antags. For example, Paradise and the original Brig Physician on Bee has it function as a sec officer (mindshielded and exempt from antag roles), while the current implementation of the Bee version makes it act as a normal role. For this PR, I did the latter and just had it function as a normal job, but if the former is preferred I can implement that.

## Equipment

When staring, they have:
- A unique jacket and hat
- Med hud glasses
- _A flash (For protection)_ Removed as requested by Anvilman6 and many others
- A roller bed
- _A mindshield As requested by Tipy_ Removed as requested by adamsong
![BPstartingequipment](https://user-images.githubusercontent.com/14363906/125684013-589ae8da-7413-4a4b-bd8f-bd635d3ccde9.PNG)

Here is a picture of them wearing their starting gear:
![bplandmark](https://user-images.githubusercontent.com/14363906/125705401-40c68a7c-9350-4c69-a384-5887beb1423b.png)

In the infirmary, there is a secure locker. within it is:

- A bag containing different outfits
- A medical PDA cartridge
- A medsec radio
- A medical belt
- A crew pinpointer
- A crew monitor
- A box of bodybags
- **An anesthetic tank** Added as requested by Anvilman6
- **A medical mask** Added as requested by Anvilman6

## Map Changes

There are numerus changes to maps, but as I have no experience with mapping, some changes might not work or could be bad. First off, I added landmarks for the brig physician (usually in the brig infirmary), but more importantly I changed most brig infirmaries. Most of the time, the changes are as follows:

- Adding a surgery table, op computer, and tools
- Adding the brig physicians locker and a blood freezer
- Adding a sleeper and medical vender (maybe a bit too much to add)
- Removing some of the morgue trays

Here is a more in depth list of changes with pictures:

Box: Basically what I said before, but with the removal of a bed to make space
![boxchanges](https://user-images.githubusercontent.com/14363906/125686547-cba5daaf-ff04-4aef-ba3c-41b63c5cefe6.png)

Meta: Has the most changed, adds a new room that is the office for the Brig Physician that replaces some of maints that contains a morgue tray, Brig Physician's locker, blood freezer, and camera console
![metachanges](https://user-images.githubusercontent.com/14363906/125687398-75157524-6884-4b19-b49b-4b15917c1e36.png)

Delta: Mainly separated the Infirmary from the brig
![deltachanges](https://user-images.githubusercontent.com/14363906/125693109-884571ee-e71d-4579-8dc9-baf985ef9750.png)

Kilo: Basically the same as the changes to box
![kilochanges](https://user-images.githubusercontent.com/14363906/125692921-80d4efd7-4e4c-4734-9780-c12f33dcd1f6.png)

## Misc Changes

- Warden has brig physician access (as it is part of the brig, they should have access to it)
- New PDA icon for the mining medic
- Mining medic starts with their jacket on (for consistency)

## Issues/To Do

- [x] Hat looks weird with the med hud glasses (grimm accepted the changes so I guess this is fine for now)
- [x] Numerus mapping issues
- [x] Add support for #11744 (added icons, just need to tweak a few vars if it gets full merged
- [x] Add skill capes
- [x] Omega and Eclipse map changes
- [x] Add support for #11973
- [x] [Wiki page](https://wiki.yogstation.net/wiki/User:Nmajask) (WIP)

# Wiki Documentation

Assuming this PR doesn't immediately get shot down, ill begin working on a wiki page for the Brig Physician on my [user page](https://wiki.yogstation.net/wiki/User:Nmajask) on the wiki, This can be transferred to its own page if it gets full merged and can act as a temporary guide if its test merged. Other than that, there are the map changes, misc changes, and a bunch of other thing that would need to be updated.

# Changelog

:cl:  
rscadd: Added the Brig Physician as a new job
tweak: Tweaked the infirmary in the brig in multiple maps
tweak: Mining medics now start with their jackets, have a custom PDA color, and have their own hat
/:cl:

nmajask committed:
Add files via upload

nmajask committed:
Merge branch 'yogstation13:master' into master

nmajask committed:
Merge branch 'yogstation13:master' into master

nmajask committed:
changed sigtech, tourist, and clerk icons

added more detail to sigtech icon
new tourist and clerk icons

nmajask committed:
Merge branch 'yogstation13:master' into master

nmajask committed:
Some more icon changes

I uploaded the wrong file

nmajask committed:
Merge branch 'yogstation13:master' into master

nmajask committed:
applied grimm-hollowknight requested changes

Adds new signal technician and paramedic hud icons and reverts clerk's back, all requested by grimm-hollowknight

nmajask committed:
ah, so thats why you reverted that change

im dumb

nmajask committed:
why cant i merge upstream

why github

nmajask committed:
Merge branch 'yogstation13:master' into master

nmajask committed:
Adds brig physicians

Adds brig physicians as a new job. There are a lot of changes, but in short:
- Added the job and many required things for it to work
- Edited maps to add lockers, landmarks, and update the brig medbay
- Added new clothing, PDAs, IDs, ect.

nmajask committed:
Fixed some stuff

- Fixed some stuff I forget to change on some maps
- Added new landmark icons

nmajask committed:
Removed Brig Physician's ability to be an antag

Removed Brig Physician's ability to be antag and gave them a mindshield

nesisknesis reviewed (approved):
switzerland

nmajask committed:
Many requested changes

- Removed the flash from the Brig Physician's starting equipment
- Removed security access for non-skeleton crew and weapon access from the Brig Physician
- Added skillcapes
- Updated the Infirmary in eclipse and omega
- Added a wall mounted flash on box
- Added an anesthetic tank and medical mask to the Brig Physician's locker
- Added icons for potential future support for the changes in #11744

nmajask committed:
Another bunch of changes

- Fixed a bunch of typos and minor mistakes (thanks TheGamerdk for pointing most of them out)
- Added support for #11744 now that its merged
- Fixed a merge conflict

nmajask committed:
Merge branch 'master' into brigphys

nmajask committed:
Might fix one of the merge conflicts

ahhhhhhhhhhhhhhh
I had to redo ALL of the map changes, so that was fun

nmajask committed:
Merge branch 'master' of https://github.com/yogstation13/Yogstation into brigphys

nmajask committed:
Should finally bring this back to a functional state

Should fix some of the checks failing cause the Kilo file was messed up
- Re-did the changes to Kilo
- Fixed something I left in while testing stuff

nmajask committed:
Thought I fixed this

- Fixes the Brig Physician's skill capes using the wrong job

patpol4 reviewed (changes requested):
### Sorry for the delay, I've been putting this off because i was ~~busy~~ lazy, here's the review:
## Eclipse:

- cool but i don't get why you put the other spawn down in normal brig, no biggie
- also the wallmed var mentioned later

## Kilo:

- please mapmerge or edit it in fastdmm2 so it mapmerges by itself

## Omega:
![vivaldi_HT7KOtMwF4](https://user-images.githubusercontent.com/48154165/127542934-b6890910-470d-4884-a329-f7388c6dfce6.png)

1. this decal looks off, just place one in the corner and remove the other two on this tile
2. (applies to all maps) why did you use a no power var on the wallmeds? Remove it unless you have some solid reason
3. you forgot firelocks in the door
4. new rooms separated by doors and firelocks need a vent and scrubber
5. keep in mind the access on the door means the medic needs perma access

## Box

- Looks like you straight up just deleted some items and didnt place them anywhere else, please restore them, could even be in some other places in the brig

## Delta

- ![image](https://user-images.githubusercontent.com/48154165/127545798-0256f9f6-6292-4676-ada9-2da39f57a785.png)
Stinky floating apc, remember to use dirs not pixelshifts for these

- There's probably not enough light in here, it's better if you use fixtures

## Meta
![vivaldi_7WVPncZz79](https://user-images.githubusercontent.com/48154165/127548701-bf328168-85e8-435f-8bba-2974675a3bf7.png)

1. apc
2. just make this the brig area

Overall pretty good for a new mapper

nmajask committed:
Should fix wejengin2's issues with the maps

Should fix most of wejengin2's issues with the map changes.
- Changes all the emergency wall meds to normal wall meds
- Might have fixed the APCs, but I don't know if I did it right
- MapMerged Kilo
- In Omega, I fixed the decals, added firelocks, and added vents/scrubbers in the brig infirmary
- In Box, I re-added some of the missing items
- In Delta, rearranged the lights and replaced them with tubes
- In Meta, I moved the morgue tray into the unused space, moved the blood freezer into the office, and added a shelf in the infirmary

nmajask committed:
Merge branch 'master' of https://github.com/yogstation13/Yogstation into yogstation13-master

nmajask committed:
Merge branch 'yogstation13-master' into brigphys
no

nmajask committed:
Merge pull request #3 from yogstation13/master

Updates the branch

nmajask committed:
Should fix Kilo

I redid all of the changes to Kilo

patpol4 reviewed (approved):
looks good to me now

nmajask committed:
Merge branch 'master' of git://github.com/yogstation13/Yogstation into brigphys

nmajask committed:
Fixes an issue with the upstream merge

- For some reason the syndie icon change didnt save, so I fixed it
- Updated the digialt stuff

nmajask committed:
Merge branch 'master' of git://github.com/yogstation13/Yogstation into brigphys

nmajask committed:
Fixes the zombie hud sprite

- Re-added the zombie hud icon by Cauckles

adamsong reviewed (dismissed):
Make the brig infirmary locked behind medical access with red alert access, and make them only report to the CMO

nmajask committed:
Adds some of adamsong's requested changes

nmajask committed:
Other part of adamsong's request

Makes it so the brig physician only reports to the CMO

nmajask committed:
Merge branch 'brigphys' of https://github.com/nmajask/Yogstation-But-Worse into brigphys

nmajask committed:
This seems important

Updates the dme

nmajask committed:
Messes with access

Made a bunch of changes, most of which were requested by adamsogm
- Removed brig phys access from the warden
-Added brig phys access to CMO and HoS
- Moved brig phys access from sec to medical
- Fixed the max skill cape

nmajask committed:
Revert "Removed Brig Physician's ability to be an antag"

This reverts commit 53cc9cb98ec43c761afcc71611b5666401d0f583.

nmajask committed:
Adds first-aid kits to sec offices

nmajask committed:
Reworks the eclipse infirmary

nmajask committed:
Adds alternate job names

Adds alternate job names, just needs to be uncommented to work (I think)

nmajask committed:
Removed changes to warden.dm

aaaaaaaah

nmajask committed:
fixes some stuff with test merges

fixes some stuff with test merges I think

nmajask committed:
Fixes more test merge conflicts

Fixes more test merge conflicts i think

nesisknesis reviewed (changes requested):
!! WOOP WOOP RED CROSS VIOLATION !!

![image](https://user-images.githubusercontent.com/37936685/129772857-773b13c8-66b3-4f29-b948-fd5e5d5ec6e4.png)

!! WOOP WOOP RED CROSS VIOLATION !!

YOU ARE IN VIOLATION OF THE GENEVA CONVENTION

nmajask committed:
Merge branch 'master' of git://github.com/yogstation13/Yogstation into brigphys

nmajask committed:
Fixes my Geneva Convention violation

Great, now my art is so bad it made me a war criminal

nmajask committed:
Merge branch 'master' of git://github.com/yogstation13/Yogstation into brigphys

nmajask committed:
Revert "Merge branch 'master' of git://github.com/yogstation13/Yogstation into brigphys"

This reverts commit 2bde593670da3537d08f743427cdd3ad4dcf6b74, reversing
changes made to f7178ffb84f4428538f4d98f11a207f3f81de0dd.

nmajask committed:
Revert "fixes some stuff with test merges"

This reverts commit 565fb9c055dd2c67281775b3695b6e664cf68bf3.

nmajask committed:
Revert "Revert "Merge branch 'master' of git://github.com/yogstation13/Yogstation into brigphys""

This reverts commit 8fcb7e0b1820dbd4f5f2aac1da240bec88ceff8f.

nmajask committed:
should fix some stuff with job alt titles

should fix some stuff with job alt titles

nmajask committed:
Should fix this mess

Im so good at coding

nmajask committed:
Fixes more things

- Fixed Brig Physician incorrectly be labeled as sec in many parts of the code
- Fixed mining medic's cap being invisable
- May have fixed the alternate job titles (I forgot to uncomment it)

nmajask committed:
Plasmameme outfit

Adds plasmameme outfit

nmajask committed:
Merge branch 'master' of git://github.com/yogstation13/Yogstation into brigphys

nmajask committed:
Rearranges some stuff in the box infirmary

nmajask committed:
Revert "Revert "Removed Brig Physician's ability to be an antag""

This reverts commit 364395971dbbb5998c49845913e22ce9facb7fae.

nmajask committed:
Fixes meta

nmajask committed:
Merge branch 'master' of git://github.com/yogstation13/Yogstation into brigphys

JamieD1 committed:
Merge branch 'master' into brigphys

adamsong committed:
Fix merge error
